### PR TITLE
Us06 talk module functionality_VladimirZelinskiy

### PR DESCRIPTION
**Pull request by VladimirZelinskiy**
Please review my final commits before merging with master branch. Thank you!

### CODE REVIEW

**Comment by VladimirZelinskiy** [comment]
@Kallen04 ready for review :)

File: src/test/resources/features/US06_TalkModuleFunctionality.feature [hunk 1]
```diff
@@ -6,22 +6,28 @@ Feature: Default
     When user enters the username "user_username" and password "user_password"
     Then user should be able to login
 
-    @B30G3-155
+  @B30G3-155
```
Comment (Kallen04) [inline src/test/resources/features/US06_TalkModuleFunctionality.feature:9]: Based on Jira, the tag of this Scenario should be B30G3-220

File: src/test/resources/features/US06_TalkModuleFunctionality.feature [hunk 2]
```diff
@@ -6,22 +6,28 @@ Feature: Default
...
-        When user clicks 3 dots icon next any conversation
-        And user clicks delete conversation button
-        And user clicks yes button
-        Then conversation should be deleted
+      | ConversationName |
+      | ABC              |
+      | DEF              |
+      | GHI              |
+
+  @B30G3-155
```
Comment (Kallen04) [inline src/test/resources/features/US06_TalkModuleFunctionality.feature:26]: The tag of this scenario should be B30G3-222

**Review by Kallen04 — APPROVED** [review]
I added some comments in your feature file. Basically the  tags  of your scenarios should be numbers of your TC that Jira assigned when you created them. 
I will approve, but please make these changes and commit again.

**Pull request merged**